specs/deneb/p2p-interface.md

```diff
@@ -241,7 +251,7 @@ No more than `MAX_REQUEST_BLOCKS_DENEB` may be requested at a time.
 
 Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
 
-[1]: # (eth2spec: skip)
+[0]: # (eth2spec: skip)
```

Comment:
I'm a bit curious when to use [0] and when to use [1] here. cc: @hwwhww

Reply:
Likewise, it's inconsistent inside the same document. Overall, [0] seems to be more.. "dominant" throughout the codebase, for these table annotations.

Reply:
Agreed. It shouldn't really matter either way, it's only used to tell pyspec not to parse the table.